edx_proctoring/views.py

```diff
@@ -763,6 +763,18 @@ def get(self, request, course_id):
             for onboarding_profile in onboarding_profile_info:
                 obscured_id = onboarding_profile['user_id']
                 user = obscured_user_ids_to_users.get(obscured_id)
+
+                if not user:
+                    log_message = (
+                        'User with obscured user ID {obs_id} not in list of learners enrolled in course '
+                        '{course_id} with proctoring eligible enrollments.'
```

Comment:
how many of these does the proctoring system return at a time? Will this be hundreds of duplicate log lines in a big mass? It might make sense to log a count or log a count plus a few examples.

Reply:
That's a good point. I don't really know. Theoretically, we didn't expect for this case to happen at all, so it's tough to guess how many logs this might generate. I can see what I can do about just logging a few examples.